src/disputables/utils.js

```diff
@@ -1,6 +1,11 @@
-import { getNetworkType } from '../lib/web3-utils'
```

Comment:
Wondering if we should rename this file differently, since it's just related to getting subgraph urls. Perhaps `connect-endpoints`?